README.md

```diff
@@ -49,6 +49,59 @@ This project uses the [pyrex](https://github.com/garmin/pyrex) tool to transpare
 
     <font color=lightgreen>[NI]</font> builders who are connected to the NI  corporate network should specify `-org` in their init script args, to provoke the script into adding the `ni-org.conf` snippet to your bitbake directory. External builders *should not* use `--org`.
 
+5. Build the package or packages that you want for your target. For example, to build Python, Ruby, and Apache for Zynq targets, run the following commands:
+
+        bitbake python ruby apache2
+
+    To build every package in NI's feed, run the following commands:
+
+        bitbake packagefeed-ni-core
+        bitbake --continue packagefeed-ni-extra
+
+    **NOTE** The configuration files (build/conf/*.conf) can optionally be changed to reflect the desired build settings instead of setting environment variables.
+
+    **NOTE** Building packages through OpenEmbedded can use significantdisk space, on the order of tens of gigabytes. If you are preparing a virtual machine to build images, make sure to allocate sufficient disk space.
+
+    The resulting ipk files that can be installed through opkg exist at
+the following directory:
+
+        tmp-glibc/deploy/ipk/...
+        
+6. Bitbake can transform tmp-glibc/deploy/ipk/ into a package feed when you run the following command:
+
+        bitbake package-index
+
+    You must rebuild the package-index before building images or if any package whom the images you are trying to build depends on has been changed and rebuilt.
+    
+7. Building various images
+
+    Build a safemode image by running the following commands:
+
+        bitbake nilrt-safemode-rootfs
+    
+    Build a runmode image by running the following commands:
+
+        bitbake nilrt-runmode-rootfs
```

Comment:
Better specify `nilrt-base-system-image` instead of `nilrt-runmode-rootfs` as the output of `nilrt-base-system-image` can directly be used from MAX.

Reply:
Do we advertise to external clients out of NI that they can go ahead and do that from MAX? Since this is external facing, I don't want to rely on the internal directory structure of MAX.

Reply:
We don't but I don't see how they can use the output of `nilrt-runmode-rootfs` directly either.

Maybe we can mention that `nilrt-runmode-system-image-x64.tar` that gets built from `nilrt-base-system-image` can be copied to a target, unpacked, then `tar xf data.tar.gz -C /mnt/userfs && ./postinst` can be run in lieu of provisioning from MAX?

I don't have a strong opinion here, so feel free to drop if you wish.

Reply:
If they cannot make use of the output of `nilrt-runmode-rootfs` directly, then what about "nilrt-safemode-rootfs"? If we want it to be more useful for customers, instead of rootfs, should I refer to `nilrt-base-system-image` instead and omit the rootfs? Would that make more sense?

Reply:
We want to point customers to the recipes that produce artifacts that are most useful.

In case of runmode, I see `nilrt-base-system-image` as a better choice since it also includes the `postinst` script which is required for successful provisioning (`nilrt-runmode-rootfs` doesn't include this).

In case of safemode, although customers may find `nilrt-recovery-media` to be a better choice to upgrade safemode, I think its ok to mention `nilrt-safemode-rootfs` also as it contains everything needed to upgrade an existing safemode installation (by unpacking the output archive and copying file from it into /boot/.safe/).

Reply:
I'm ok with listing nilrt-base-system-image, provided that we clearly indicate that using so "voids the warranty" on support.  That ok with everybody?

Reply:
Why does using the nilrt-base-system-image void support? I don't follow. I also prefer to have instructions to build nilrt-base-system-image instead of the rootfs.

Reply:
Updated to nilrt-base-system-image.

@jeminor What is the line between "voiding the warranty" and valid use case? If they just built our branch as it is without making changes, I am assuming that is a valid use case, is that correct? Only if they change the recipes or configuration materially. Then it is "voiding the warranty"?

Reply:
As discussed, the MIT license and the NI EULA already covered the use cases of client building from source.